sdk/anomalydetector/Azure.AI.AnomalyDetector/README.md

```diff
@@ -77,9 +109,104 @@ We guarantee that all client instance methods are thread-safe and independent of
 [Client lifetime](https://devblogs.microsoft.com/azure-sdk/lifetime-management-and-thread-safety-guarantees-of-azure-sdk-net-clients/)
 <!-- CLIENT COMMON BAR -->
 
+## Examples
+
+The following section provides several code snippets covering some of the most common Anomaly Detector service tasks, including:
+
+- [Univariate Anomaly Detection - Batch detection](#batch-detection)
+- [Univariate Anomaly Detection - Streaming detection](#streaming-detection)
+- [Univariate Anomaly Detection - Detect change points](#detect-change-points)
+- [Multivariate Anomaly Detection](#multivariate-anomaly-detection-sample)
+
+### Batch detection
+
+```csharp
```

Comment:
We use use code snippet instead of raw test for code. See examples under samples folder.